docker/php.Dockerfile

```diff
@@ -220,14 +222,15 @@ RUN wget -O /usr/local/bin/php-fpm-healthcheck https://raw.githubusercontent.com
 #============================================
 # FPM Extensions
 #============================================
-RUN docker-php-ext-enable dom && \
-    docker-php-ext-enable gd && \
-    docker-php-ext-enable igbinary && \
+RUN docker-php-ext-enable gd && \
```

Comment:
Removed `dom`, `igbinary` and `zip`, why?

Reply:
we're not using it in `php-fpm` ;-)